CHANGELOG.md

```diff
@@ -66,6 +66,7 @@ Here is an overview of all new **experimental** features:
 
 ### Fixes
 
+- **General**: Set LeaderElectionNamespace to PodNamespace ([#5404](https://github.com/kedacore/keda/issues/5404))
```

Comment:
```suggestion
- **General**: Set `LeaderElectionNamespace` to PodNamespace so leader election works in OutOfCluster mode ([#5404](https://github.com/kedacore/keda/issues/5404))
```

We are shipping 2.13, we can include this, just move it down below to 2.13, Improvements section

Reply:
Thanks, wasn't sure as 2.13 is coming out tomorrow :)

Reply:
updated in my latest commit